examples/sites_example_test.go

```diff
@@ -0,0 +1,75 @@
+//+build examples
+
+package examples
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSitesAPIExample(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(runCRUD())
```

Comment:
```suggestion
	assert.NoError(t, runCRUD())
```
Then, assert := assert.New(t) can be removed.

Reply:
Resolved